projects/canopy/src/lib/brand-icon/brand-icon.component.spec.ts

```diff
@@ -57,7 +57,11 @@ describe('LgBrandIconComponent', () => {
       fixture.detectChanges();
 
       expect(fixture.nativeElement.querySelector('#test')).toBeNull();
-      expect(fixture.nativeElement.querySelector('#lg-brand-icon-0')).toBeDefined();
+      expect(
+        /lg-brand-icon-\d-\d/.test(
```

Comment:
i've updated this to match the pattern of the id that is set to ensure ids are unique. this is related to the bug #248 . I had attempted to access the nextUniqueId to reset it in the afterEach hook but it doesn't appear to be technically global. Asserting the id has this pattern does however provide value in confirming the replace has occurred